Source/Resources/Content/PowerShell/Exchange/Organization/OrganizationConfig/First Organization.json

```diff
@@ -25,6 +25,7 @@
   "defaultPublicFolderMaxItemSize": "Unlimited",
   "defaultPublicFolderMovedItemRetention": "7.00:00:00",
   "defaultPublicFolderProhibitPostQuota": "2 GB (2,147,483,648 bytes)",
+  "description":"What is it: Defines various Exchange settings. \n Simeon Notes: Microsoft changes these setting frequently as features are added and removed. \n Learn More: https://docs.microsoft.com/en-us/powershell/module/exchange/set-organizationconfig?view=exchange-ps",
```

Comment:
Yeah @metamenos , I just added you to the DevOps org 'Simeon-Test1' which has both a first time export and deploy. Not sure how to link it so I just added you

Reply:
@metamenos this file is unchanged, see the attached screenshot 
<img width="633" alt="Screen Shot 2020-08-24 at 11 24 03 AM" src="https://user-images.githubusercontent.com/70111688/91081668-53487c00-e5fc-11ea-91bc-caaee267066d.png">

Reply:
So in terms of non-default things we are configuring, there are none. But I guess why this would be useful is to track config changes made in the environment. 

As for user impact, since we are doing only default settings there is no impact.

Reply:
@metamenos how's this? just updated